client/webserver/site/src/js/doc.ts

```diff
@@ -258,15 +259,19 @@ export default class Doc {
     return decimalFormatter(prec).format(v)
   }
 
-  static formatThreeSigFigs (v: number): string {
-    if (v >= 1000) return intFormatter.format(Math.round(v))
-    return threeSigFigs.format(v)
+  /*
+   * formatRate formats rate to represent it exactly at rate step precision,
+   * trimming non-effectual zeros if there are any.
+   */
+  static formatRateToRateStep (encRate: number, bui: UnitInfo, qui: UnitInfo, rateStepEnc: number) {
```

Comment:
We are formatting an **encoded rate** to a **conventional rate**. Can the method name reflect that? This is also full-precision, so maybe a name like `formatRateFullPrecision` would be more apt.

Reply:
> so maybe a name like formatRateFullPrecision would be more apt.

Yep, sounds good, full precision also implies that it is a **conventional** rate. [Applied](https://github.com/decred/dcrdex/pull/2278/commits/fc82f2b2fa856572a36d9ee64a0cd168bc78e0a6).